src/js/controller/model.js

```diff
@@ -236,6 +234,7 @@ define([
 
             // Attempt setting the playback rate to be the user selected value
             this.setPlaybackRate(this.get('defaultPlaybackRate'));
+            this.setPlaybackRatesAvailable();
```

Comment:
The view should control whether the playbackRate controls are shown, not the model.